org-code-javabuilder/theater/src/main/java/org/code/theater/Stage.java

```diff
@@ -420,4 +427,20 @@ private void writeImageAndAudioToFile() {
       throw new InternalJavabuilderError(InternalErrorKey.INTERNAL_RUNTIME_EXCEPTION, e);
     }
   }
+
+  private void playNote(Instrument instrument, int note, double noteLength, boolean shouldPause) {
+    final String sampleFilePath = instrumentSampleLoader.getSampleFilePath(instrument, note);
+    if (sampleFilePath == null) {
+      return;
+    }
+
+    try {
+      this.audioWriter.writeAudioFromLocalFile(sampleFilePath, noteLength);
+      if (shouldPause) {
+        this.pause(noteLength);
+      }
+    } catch (FileNotFoundException e) {
+      System.out.printf("Could not play instrument: %s at note: %s%n", instrument, note);
```

Comment:
should this throw an exception instead?

Reply:
So per curriculum, if `playNote` is called with in invalid note/instrument, we just want return and log an error instead of throwing